interface/forms/fee_sheet/review/fee_sheet_classes.php

```diff
@@ -76,16 +76,21 @@ class procedure extends code_info
     function __construct($c, $ct, $desc, $fee, $justify, $modifiers, $units, $mod_size, $selected = true)
     {
         parent::__construct($c, $ct, $desc, $selected);
-
+        /**
+         *  This call to the database is to get the current price to see if any changes have been made
+         */
         $currentPrice = sqlQuery("SELECT codes.id, prices.pr_price FROM `codes` LEFT JOIN prices ON prices.pr_id = codes.id WHERE codes.code = ?", array($c));
```

Comment:
not a biggie but prob don't need to select codes.id since only interested in pr_price

Reply:
that is true, the id does not have to be brought back. I just do that on development but can be dropped for production.